### PR TITLE
Colemak now working with special chars

### PR DESCRIPTION
**Pull request by MackanT**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/4x6/info.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
         "soft_serial_pin": "D0"
     },
     "processor": "atmega32u4",
-    "bootloader": "caterina",
+    "development_board": "promicro",
+    "pin_compatible": "promicro",
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/4x6/info.json:27]: Processor and pin compatibility is set by dev board. 
```suggestion
    "development_board": "promicro",
```

File: keyboards/handwired/dactyl_manuform/5x6_main/.gitignore
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_main/.gitignore:1]: This should be removed.

File: keyboards/handwired/dactyl_manuform/5x6_main/LICENSE
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_main/LICENSE:1]: ?

File: keyboards/handwired/dactyl_manuform/5x6_main/info.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        "rgblight": false,
+        "audio": false,
+        "backlight": false
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "D0"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "caterina",
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_main/info.json:33]:
```suggestion
    "development_board": "promicro",
```

File: keyboards/handwired/dactyl_manuform/5x6_main/keymaps/swedish/config.h
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_main/keymaps/swedish/config.h:1]: This file can be removed, as default behavior is master left.

File: keyboards/handwired/dactyl_manuform/5x6_main/keymaps/swedish/keymap.c [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+     _______, KC_BTN4, KC_BTN5, _______, _______, _______,                                          _______, _______, _______, _______, _______, _______,
+     _______, KC_BTN1, KC_MS_U, KC_BTN2, KC_WH_U, _______,                                          _______, KC_BTN1, KC_BTN3, KC_BTN2, _______, _______,
+     _______, KC_MS_L, KC_MS_D, KC_MS_R, KC_WH_D, _______,                                          _______, KC_ACL0, KC_ACL1, KC_ACL2, _______, _______,
+     _______, _______, _______, _______, _______, _______,                                          _______, _______, _______, _______, _______, _______,
+                       _______, _______, _______, _______, _______, _______,      _______, _______, _______, _______, _______, _______,
+                                         _______, _______,                                          _______, _______
+  ),
+};
+
+// //Determine the current tap dance state
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/5x6_main/keymaps/swedish/keymap.c:111]: Commented code should be removed.

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by MackanT** [comment]
Opps, no it was not meant for the main repository, my bad!

**Pull request closed without merging**